docs/reference/releases/rev156_126.md

```diff
@@ -0,0 +1,60 @@
+# Revision 156/126
+<sub>Wednesday, February 28, 2024</sub>
+
+Dear community,
+
+We are extremely thrilled and excited to share that Charmed Kafka and Charmed ZooKeeper have now been released as GA. You can find them in [charmhub.io](https://charmhub.io/) under the `3/stable` track.
+
+More information are available in the [Canonical website](https://canonical.com/data/kafka), alongside its [documentation](https://canonical.com/data/docs/kafka/iaas).
+Also find the full announcement of the release [here](https://canonical.com/blog/charmed-kafka-general-availability) and [here](https://discourse.charmhub.io/t/announcing-general-availability-of-charmed-kafka/13277). 
+And more importantly, make sure you don't miss out the [webinar](https://www.linkedin.com/events/7161727829259366401/about/) that Raúl Zamora and Rob Gibbon will be holding later today.
```

Comment:
```suggestion
The full version of the announcement can be seen in our [blog](https://canonical.com/blog/charmed-kafka-general-availability) and [forum](https://discourse.charmhub.io/t/announcing-general-availability-of-charmed-kafka/13277). 
For more information on Charmed Kafka, see the [Canonical website](https://canonical.com/data/kafka), [documentation](https://canonical.com/data/docs/kafka/iaas), and our webinar [Introduction to Apache Kafka](https://www.linkedin.com/events/7161727829259366401/about/) by Raúl Zamora and Rob Gibbon.
```
Some suggestions on wording.
Also, the webinar announcement should be more specific on the topic and less bound to the current date (ideally, we want it to stay correct before, during, and after the webinar).

Reply:
as above